NuGet.config

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="True" />
+  </solution>
+  <packageRestore>
+    <add key="enabled" value="True" />
+    <add key="automatic" value="True" />
+  </packageRestore>
+  <packageSources>
+    <add key="Official source v3" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
```

Comment:
I have never needed this. The docs for [dotnet restore](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore#specify-feeds) says:

> To restore the dependencies, NuGet needs the feeds where the packages are located. Feeds are usually provided via the nuget.config configuration file. A default configuration file is provided when the .NET SDK is installed. To specify additional feeds, do one of the following:

So I don't think this is needed.